activemodel/test/cases/callbacks_test.rb

```diff
@@ -55,33 +55,33 @@ def create
   test "complete callback chain" do
     model = ModelCallbacks.new
     model.create
-    assert_equal model.callbacks, [ :before_create, :before_around_create, :create,
-                                    :after_around_create, :after_create, :final_callback]
+    assert_equal([ :before_create, :before_around_create, :create,
+                                    :after_around_create, :after_create, :final_callback], model.callbacks)
   end
 
   test "the callback chain is not halted when around or after callbacks return false" do
     model = ModelCallbacks.new
     model.create
-    assert_equal model.callbacks.last, :final_callback
+    assert_equal(:final_callback, model.callbacks.last)
   end
 
   test "the callback chain is not halted when a before callback returns false)" do
     model = ModelCallbacks.new(before_create_returns: false)
     model.create
-    assert_equal model.callbacks.last, :final_callback
+    assert_equal(:final_callback, model.callbacks.last)
   end
 
   test "the callback chain is halted when a callback throws :abort" do
     model = ModelCallbacks.new(before_create_throws: :abort)
     model.create
-    assert_equal model.callbacks, [:before_create]
+    assert_equal([:before_create], model.callbacks)
   end
 
   test "after callbacks are not executed if the block returns false" do
     model = ModelCallbacks.new(valid: false)
     model.create
-    assert_equal model.callbacks, [ :before_create, :before_around_create,
-                                    :create, :after_around_create]
+    assert_equal([ :before_create, :before_around_create,
+                                    :create, :after_around_create], model.callbacks)
```

Comment:
Indentation is a bit odd here too.  How about:
```suggestion
    assert_equal \
      [:before_create, :before_around_create, :create, :after_around_create],
      model.callbacks
```